lib/caasp.pm

```diff
@@ -174,9 +174,10 @@ sub update_scheduled {
 
     # Filter for update types
     return $repo unless $type;
-    return $repo =~ 'Maintenance' if $type eq 'qam';
```

Comment:
has to be case sensitive ?

Reply:
It's checking `http://download.suse.de/ibs/SUSE:/Maintenance:/10277/SUSE_Updates_SUSE-CAASP_3.0_x86_64/`
I can leave this case sensitive.

Reply:
I don't know and I don't mind, but I have seen the change, just curios if it has some reason. :smiley: